capstone/src/components/User/Adventurer/FindAdventure/FindAdventure.jsx

```diff
@@ -0,0 +1,34 @@
+import './FindAdventure.css';
+import { useState } from 'react';
+import Map from './Map/Map';
+
+export default function FindAdventure() {
+  const [displayExperience, setDisplayExperience] = useState('map');
+  return (
+    <div className="findAdventure">
+      <h2>Find your next adventure:</h2>
+      <ExperienceDisplayer
+        displayExperience={displayExperience}
+        setDisplayExperience={setDisplayExperience}
+      />
+      <Map />
+    </div>
+  );
+}
+
+export function ExperienceDisplayer({ displayExperience, setDisplayExperience }) {
```

Comment:
Instead of passing a setter *down*, try passing the callback *up*.

This article on [controlled components](https://reactjs.org/docs/forms.html#controlled-components) might be helpful.

```suggestion
export function ExperienceDisplayer({ displayExperience, onSelectDisplay }) {
```